pkg/builder/builder.go

```diff
@@ -567,3 +616,80 @@ func HasProhibited(tags []string) bool {
 
 	return false
 }
+
+func ContainsActions(target string, sas []schemas.ScheduledAction) bool {
+	for _, sa := range sas {
+		if sa.Name == target {
+			return true
+		}
+	}
+	return false
+}
+
+// ValidCronExpression checks if the cron expression is valid or not
+// It should be [Minute] [Hour] [Day_of_Month] [Month_of_Year] [Day_of_Week]
+func ValidCronExpression(expression string) (bool, error) {
+	elements := strings.Split(expression, " ")
+	if len(elements) != 5 {
+		return false, fmt.Errorf("cron expression should be in the format of [Minute] [Hour] [Day_of_Month] [Month_of_Year] [Day_of_Week]: %s", expression)
+	}
+
+	// minutes
+	if len(elements[0]) > 0 && elements[0] != "*" {
+		i, _ := strconv.Atoi(elements[0])
+		if i < 0 || i > 60 {
+			return false, fmt.Errorf("first element should be from 0 to 60 or *: %s", expression)
```

Comment:
May be it would be 0 to 59, isn't it?

Reply:
I don't know exactly about ASG's recurrence rules.

But crontab spec is
```
           The time and date fields are:

                    field          allowed values
                    -----          --------------
                    minute         0-59
                    hour           0-23
                    day of month   0-31
                    month          0-12 (or names, see below)
                    day of week    0-7 (0 or 7 is Sun, or use names)
```
ref: http://crontab.org/

Reply:
Oh this should be fixed..! 👍